components/frontend/src/metric/MetricDetails.test.jsx

```diff
@@ -101,12 +101,12 @@ async function renderMetricDetails(stopFilteringAndSorting, connection_error, fa
     let result
     await act(async () => {
         result = render(
-            <LocalizationProvider dateAdapter={AdapterDayjs} adapterLocale={locale_en_gb}>
```

Comment:
The locale no longer needed, or automagic?

Reply:
If no adapterLocale is passed it uses the default (en-US). Since we're not testing a specific locale here it doesn't make sense to pass one.